x-pack/legacy/plugins/lens/public/xy_visualization_plugin/_xy_config_panel.scss

```diff
@@ -1,3 +1,19 @@
-.lnsXyConfigPanel-layer {
-  border: 1px solid $euiColorLightestShade;
+.lnsConfigPanel__panel {
+  margin-bottom: $euiSizeS;
+}
+
+.lnsConfigPanel__axis {
+  background: $euiColorLightestShade;
+  padding: $euiSizeS;
+  border-radius: $euiBorderRadius;
+
+  & + & {
```

Comment:
This will be unfamiliar for a lot of folks. Might be worth a comment.

Reply:
```suggestion
  // Add margin to the top of the next same panel
  & + & {
```